static/app/views/performance/transactionSummary/transactionSpans/spanDetails/exclusiveTimeHistogram.tsx

```diff
@@ -142,99 +107,63 @@ type ChartProps = {
   isErrored: boolean;
   isLoading: boolean;
   location: Location;
-  colors?: string[];
+  zoomProps: any;
   disableChartPadding?: boolean;
-  disableXAxis?: boolean;
-  disableZoom?: boolean;
-  grid?: BarChart['props']['grid'];
-  height?: number;
 };
 
 export function Chart(props: ChartProps) {
-  const {
-    isLoading,
-    chartData,
-    location,
-    height,
-    grid,
-    disableXAxis,
-    disableZoom,
-    colors,
-  } = props;
+  const {chartData, zoomProps} = props;
 
   if (!chartData) {
-    return null;
+    return <Placeholder height="200px" />;
   }
-  const theme = useTheme();
 
-  const series = {
-    seriesName: t('Count'),
-    data: formatHistogramData(chartData, {type: 'duration'}),
-  };
+  const theme = useTheme();
 
-  const xAxis = {
-    type: 'category' as const,
-    truncate: true,
-    boundaryGap: false,
-    axisTick: {
-      alignWithLabel: true,
+  const chartOptions = {
+    grid: {
+      left: '10px',
+      right: '10px',
+      top: '40px',
+      bottom: '0px',
+    },
+    colors: theme.charts.getColorPalette(1),
+    seriesOptions: {
+      showSymbol: false,
     },
+    tooltip: {
+      trigger: 'axis' as const,
+      valueFormatter: (value, _seriesName) => tooltipFormatter(value, 'integer'),
```

Comment:
`integer` isn't correct, it'll just so happen to be `number` because it fails to find it as a series name... we really should fix these to pull from meta instead. We don't really do this correct everywhere so we should be fixing it to stop coercing it like this, but we can defer that out of this PR. 

Just use the yAxis even if it's incorrect for now.

Reply:
ah I see. I'll pull it from the meta in the follow up PR 👍